release notes/v0.53.0.md

```diff
@@ -0,0 +1,230 @@
+k6 `v0.53.0` is here 🎉! This release includes:
+
+- Native ECMAScript modules support :tada:!
+- New experimental OpenTelemetry metrics output :rocket:!
+- Blob support in experimental websockets module
+- Consolidating cloud features and commands under `k6 cloud` :cloud:
+- Breaking change: remove magic URL resolutions :stop_sign:
+
+## Breaking changes
+
+### Require is now specification compliant and always resolve based on the file it is written in [#3534](https://github.com/grafana/k6/issues/3534)
+
+`require` in k6 used to resolve identifiers based on the current "root of execution"(more on that later). In a lot of cases that aligns with the file the `require` is written in or a file in the same folder, which leads to the same result. In small subset of cases this isn't case.
+
+Every other implementation and more or less by the CommonJS specification - require should always be relative to the file it is written in.
+
+This also aligns with how ESM and dynamic `import` also work. In order to align with them `require` now uses the same underlying implementation.
+
+There was a warning message for the last 2 releases trying to tease out cases where that would be problematic.
+
+<details>
+  <summary>"root of exeuction" explanation</summary>
+This is very much an implementation detail that has leaked and likely a not intended one.
+
+Whenever a file is `require`-ed it become the "root of execution" and both `require` and `open` become relative to it. Once the `require` finishes, the previous "root of execution" gets restored. During not init context execution, the main file is the "root of execution".
+
+Example:
+
+Have 3 files:
+main.js
+```javascript
+const s = require("./A/a.js")
+if (s() != 5) {
+	throw "Bad"
+}
+module.exports.default = () =>{} // just for k6 to not error
+```
+
+/A/a.js:
+```javascript
+module.exports =  function () {
+  return require("./b.js");
+}
+```
+/A/b.js
+```javascript
+module.exports = 5
+```
+In this example when `require` is called in `/A/a.js` the `main.js` is once again the "root of execution". If you call the function in `/A/a.js` just after defining it though, it will work as expected.
+
+</details
+
+In order to facilitate being able to write helpers that use `require` (or [dynamic import](https://github.com/grafana/k6/issues/3859) and `open` in the future) there is currently [proposal](https://github.com/grafana/k6/issues/3856) to add `import.meta.resolve` to k6.
+
+### ECMAScript Modules(ESM) Native Support related breaking changes:
```

Comment:
```suggestion
### ECMAScript Modules (ESM)
```

I don't think we need to detail it in the title. The text after does already.

Reply:
I prefer to have it as there are quite a few breaking changes and arguably ESM isn't a breaking change - there are breaking changes due to it no being native.